src/Listener/CodeCoverageListener.php

```diff
@@ -165,8 +133,6 @@ public function beforeSuite(SuiteEvent $event): void
     public static function getSubscribedEvents(): array
     {
         return [
-            'beforeExample' => ['beforeExample', -10],
-            'afterExample' => ['afterExample', -10],
```

Comment:
Without these example events the reported coverage drops to 0%. So I guess they are actually doing something (although I don't know what...).

Reply:
Yep you are right, let close this PR for the moment (I've another solution for my project trouble ^^).